deploy/helm/flightctl/templates/flightctl-api-config.yaml

```diff
@@ -46,6 +46,7 @@ data:
     kv:
         hostname: flightctl-kv.{{ default .Release.Namespace .Values.global.internalNamespace }}.svc.cluster.local
         port: 6379
+        password: {{ .Values.kv.password }}   # we should funnel this via secrets instead
```

Comment:
I agree, if we cannot do it now, can we track it somehow in JIRA?

Reply:
we have the same issue with database, probably both should be on the same JIRA.

Reply:
I know, rabbitmq as well.  That's why I wanted to level set valkey first, then fix them all.